lectures/greek_square.md

```diff
@@ -18,11 +18,21 @@ kernelspec:
 
 ## Introduction
 
-This lectures provides an  example of **invariant subspace** methods for analyzing linear difference equations. 
 
-These methods are applied throughout applied economic dynamics, for example, in this QuantEcon lecture {doc}`money financed government deficits and inflation <money_inflation>`
+This lecture can be viewed as a sequel to this QuantEcon lecture {doc}`eigenvalues and eigenvectors <eigen_I>` 
```

Comment:
If using the title from the doc then we should just use:

```
{doc}`eigen_I.md`
```

which will adopt the title from the other document